osf/models/preprint.py

```diff
@@ -1086,7 +1086,7 @@ def update_why_no_data(self, auth: Auth, why_no_data: str, log: bool = True, sav
         if self.why_no_data == why_no_data:
             return
 
-        if not self.has_data_links == 'available':
+        if not self.has_data_links == 'no':
```

Comment:
so if `not self.has_data_links == 'no':` shouldn't this be `self.has_data_links == 'no': without the `not`?